app/controllers/int_api/marketplaces_controller.rb

```diff
@@ -46,8 +46,12 @@ def create
         locale: params[:marketplace_language]},
         marketplace[:id]).data
 
+    # make the marketplace creator land to the admin panel
+    url = "#{marketplace[:url]}/admin"
```

Comment:
To me it feels that this is the right layer 👍 

However, we should use the path helpers instead of hard-coding the URLs (i.e. the hard-coded `/admin` URL here). AFAIK path helpers can take an additional `host` parameter.